jmh-tck/src/main/java/org/hibernate/performance/search/tck/AutomaticIndexingStateHolder.java

```diff
@@ -0,0 +1,51 @@
+package org.hibernate.performance.search.tck;
+
+import java.util.Properties;
+
+import org.hibernate.performance.search.model.application.ModelService;
+import org.hibernate.performance.search.model.application.ModelServiceFactory;
+import org.hibernate.performance.search.model.asset.AutomaticIndexingState;
+import org.hibernate.performance.search.model.param.RelationshipSize;
+
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.infra.ThreadParams;
+
+@State(Scope.Benchmark)
+public abstract class AutomaticIndexingStateHolder {
+
+	@Param({ "SMALL" })
+	private RelationshipSize relationshipSize;
+
+	@Param({ "100" })
+	private int initialIndexSize;
```

Comment:
I think this should be `initialCompanyCount`, like in `MassIndexingPerformanceTest` and `SearchingPerformanceTest`?